spec/plugins/data_table/local.js

```diff
@@ -294,10 +294,12 @@ describe("DataTable Plugin", function() {
         // we need to test this using an async strategy because the checkbox is toggled async as well
         table.dataTable.on("page", function() {
           _.defer(function() {
-            table.page("previous");
             expect(table.$("th.bulk :checkbox").prop("checked")).toEqual(true);
+            done();
```

Comment:
this test was not correct before, and was throwing warnings in `grunt spec`, now it's fixed